src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,22 +2,29 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("login was called, login: {}", login);
+        System.out.println("login");
```

Comment:
System.out.println statements should be removed before submitting the solution as they are typically used for debugging purposes and do not provide meaningful information to the end user or in the application logs.